superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.test.tsx

```diff
@@ -39,6 +39,14 @@ const store = mockStore({
   common: { locale: 'en' },
 });
 
+// case when common.locale is not populated
+const mockEmptyStore = configureStore([thunk]);
+const emptyStore = mockEmptyStore({});
+
+// case when common.locale is populated with invalid locale
+const mockInvalidStore = configureStore([thunk]);
+const invalidStore = mockInvalidStore({ common: { locale: 'invalid_locale' } });
```

Comment:
I'm not familiar with the return object of configureStore but can you create two separate distinct stores with the same return of `configureStore`? In other words, can we create empty and invalid both from the `mockEmptyStore`?

Reply:
Great catch!  I was able to consolidate one call to configureStore, and reuse calls to mockStore with the different values.